### PR TITLE
Changed mins into hours in const hour

### PR DESCRIPTION
**Pull request by ChielHackman**
Hi Wes.

I felt free to fix a small typo/bug. Also send you an email about this a few minutes ago.

### CODE REVIEW

**Comment by dandenney** [comment]
I already forked this and have commits for my dailies, but I believe the divisor for hours needs to be 24 as well.

`const hourDegrees = ((hours / 12) * 360) + 90;`

**Comment by ManInBoat** [comment]
hey dandenney , but  where is the hourhand's position when it  is twelve o'clock?  🙂

File: 02 - JS + CSS Clock/index-FINISHED.html [hunk 1]
```diff
@@ -85,7 +85,7 @@
     minsHand.style.transform = `rotate(${minsDegrees}deg)`;
 
     const hour = now.getHours();
-    const hourDegrees = ((mins / 12) * 360) + 90;
+    const hourDegrees = ((hours / 12) * 360) + 90;
```
Comment (fabricionaweb) [inline 02 - JS + CSS Clock/index-FINISHED.html]: `hours` or `hour` ?

**Comment by dandenney** [comment]
@ManInBoat Yeah, you're right. I popped in 12 and it points to 6 at that point.

**Comment by wesbos** [comment]
Fixed in #49

**Pull request closed without merging**